changelog.rst

```diff
@@ -0,0 +1,970 @@
+   `[#459] <https://github.com/BAMWelDX/weldx/pull/459>`__,
+   `[#469] <https://github.com/BAMWelDX/weldx/pull/469>`__.
+-  Merging and unmerging multiple `CoordinateSystemManager` instances
+   now correctly preserves all attached data.
+   `[#494] <https://github.com/BAMWelDX/weldx/pull/494>`__.
+-  `compare_nested` can compare sets
+   `[#496] <https://github.com/BAMWelDX/weldx/pull/496>`__
+
+documentation
+~~~~~~~~~~~~~
+
+-  added installation guide with complete environment setup (Jupyterlab
+   with extensions) and possible problems and solutions
+   `[#450] <https://github.com/BAMWelDX/weldx/pull/450>`__
+-  split API documentation into user classes/functions and a full API
+   reference `[#469] <https://github.com/BAMWelDX/weldx/pull/469>`__.
+
+ASDF
+~~~~
+
+-  all schema version numbers set to `0.1.0`
+   `[#535] <https://github.com/BAMWelDX/weldx/pull/535>`__.
+-  add `time/time` schema to support `Time` class
+   `[#463] <https://github.com/BAMWelDX/weldx/pull/463>`__.
+-  rework ASDF extension to new asdf 2.8 API
+   `[#467] <https://github.com/BAMWelDX/weldx/pull/467>`__
+
+   -  move schema files to `weldx/schemas`
+   -  create extension manifest in `weldx/manifests`. The manifest
+      also contains tag mappings for legacy tag names for backwards
+      compatibility.
+   -  move tag module to `weldx/tags`
+   -  refactor all asdf uris to new `asdf://` naming convention, see
+      https://asdf.readthedocs.io/en/latest/asdf/extending/uris.html#entities-identified-by-uri
+   -  replaced all referenced weldx tag versions in schemas with
+      `0.1.*`
+   -  refactor
+      `asdf://weldx.bam.de/weldx/schemas/datamodels/single_pass_weld-1.0.0.schema`
+      to
+      `asdf://weldx.bam.de/weldx/schemas/datamodels/single_pass_weld-0.1.0`
+      and enable schema test
+   -  add legacy class for validators support in
+      `weldx.asdf._extension.py`
+   -  asdf utility functions `weldx.asdf.util.uri_match`,
+      `weldx.asdf.util.get_converter_for_tag` and
+      `weldx.asdf.util.get_weldx_extension`
+   -  add `devtools/scripts/update_manifest.py` to auto update
+      manifest from extension metadata
+   -  custom shape validation must now be implemented via staticmethod
+      `WeldxConverter.shape_from_tagged`
+
+-  provide legacy schema support in
+   `weldx/schemas/weldx.bam.de/legacy`
+   `[#533] <https://github.com/BAMWelDX/weldx/pull/533>`__
+-  rewrote
+   `asdf://weldx.bam.de/weldx/schemas/core/transformations/coordinate_system_hierarchy`
+   schema for the `CoordinateSystemManager`. It uses the digraph
+   schemas to serialize the coordinate system structure.
+   `[#497] <https://github.com/BAMWelDX/weldx/pull/497>`__
+-  add `asdf://weldx.bam.de/weldx/schemas/unit/quantity` and
+   `asdf://weldx.bam.de/weldx/schemas/unit/unit` schemas
+   `[#522] <https://github.com/BAMWelDX/weldx/pull/522>`__
+
+deprecations
+~~~~~~~~~~~~
+
+dependencies
+~~~~~~~~~~~~
+
+0.4.1 (20.07.2021)
+------------------
+
+.. _added-1:
```

Comment:
does something break if we delete these links?

Reply:
dont think so. I've removed them